### ISSUE TITLE
Magazine view

### ISSUE THREAD

**Issue by VlaK0r** [Athou/commafeed#1100]
Hi! Thanks for the wonderful work.
It'd be nice to have a magazine view for feeds with pictures. Each entry would have a thumbnail and next to that the title and the first 200 characters of the description or content.

**Comment by Athou** [comment]
Why not, althought that only works for feeds that have images. What about those whithout?

**Comment by VlaK0r** [comment]
> Why not, althought that only works for feeds that have images. What about those whithout?

There are other viewing modes for channels without images. Ideally, it would be possible to make the viewing mode for each channel be saved separately

**Comment by xekil** [comment]
+1000 add feature please :)

**Comment by amboutwe** [comment]
> Why not, althought that only works for feeds that have images. What about those whithout?

A placeholder image is how other platforms handle no images. Or you can leave the image markup out entirely if you go with a card or masonry layout.

Card view
![card_view_mixed_images](https://github.com/Athou/commafeed/assets/8060612/2642559f-1b77-4f13-bb6d-6f0f4add3b22)

Magazine view
![magazine_view_mixed_images](https://github.com/Athou/commafeed/assets/8060612/5ad5cc11-c621-40f3-a217-a9c570206ed6)

**Comment by fccfc5d245c** [comment]
Yes, I just discovered Commafeed, I love it and I'm switching to it, thank you very much, but this feature would make my life a lot easier: I follow a huge amount of artists, designers and image resources, and those are best browsed by looking at the thumbnails, only reading the titles after an image draws your attention.

I'm sure there are more use cases where magazine mode is actually useful and not just making the software look pretty.

In fact, I might have to reluctantly keep an inferior but more visual RSS reader just for those sources, that's how critical this is to me, so a big +1 to the proposed enhancement.

**Comment by xekil** [comment]
please add feature

**Comment by Athou** [comment]
Would something like that work?

![image](https://github.com/user-attachments/assets/d192e637-7db4-42ba-94ad-8e87b82396a2)

**Comment by xekil** [comment]
> Would something like that work?
> 
> ![image](https://private-user-images.githubusercontent.com/1256795/371448012-a62ae487-949d-4e13-affb-d50d4d06cc11.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc0MjQ4MzQsIm5iZiI6MTcyNzQyNDUzNCwicGF0aCI6Ii8xMjU2Nzk1LzM3MTQ0ODAxMi1hNjJhZTQ4Ny05NDlkLTRlMTMtYWZmYi1kNTBkNGQwNmNjMTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjdUMDgwODU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2I5NzBjOWZjZWFkMTVhM2U3YTM4YzM5ODNmMTY3OTdlM2Y5OTIwZTczNGU4MzcwZmU0ODhjNDQ5NzJmYjQwNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.xeS1FsCzjuyqqUUA8_aZtgPDNVXojTF561xSepZGTJM)

oh yes i think that would be perfect :)

**Comment by xekil** [comment]
the ideal would be for the thumbnail to occupy the entire height and for the title to pass over the summary

**Comment by xekil** [comment]
oh yes i think that would be perfect :)
the ideal would be for the thumbnail to occupy the entire height and for the title to pass over the summary
This would really be perfect.
![feed](https://github.com/user-attachments/assets/59516613-e7c9-4f1e-a956-2abc3c27ac92)

**Comment by xekil** [comment]
Here is a picture of what would be really perfect
![feed](https://github.com/user-attachments/assets/59516613-e7c9-4f1e-a956-2abc3c27ac92)

**Comment by fccfc5d245c** [comment]
I like xekil's larger images, but both mockups would work for me. Whatever is easier to implement  :)

**Comment by xekil** [comment]
@Athou Has there been any news on the possible addition of this feature? Thanks

**Comment by Athou** [comment]
Not yet, I still need to find the time to add this (and the motivation because I will probably not use this layout myself).

**Comment by xekil** [comment]
Hello, ok thank you for you reply

**Comment by valuex** [comment]
> Would something like that work?
> 
> ![image](https://github.com/user-attachments/assets/a62ae487-949d-4e13-affb-d50d4d06cc11)
> 
> 
Basically,it is.
If is configurable for the users to set different mode, that would be more awesome.

* Magazine Mode1: small pic view on the left
![Screenshot_20250301_224424_com.microsoft.emmx.jpg](https://github.com/user-attachments/assets/276a40c8-1bf6-43ca-942a-5c839aa74fba)
* Magazine Mode2: small pic view on the right

* Magazine Mode3: big pic view at the bottom

![Screenshot_20250301_225812_com.microsoft.emmx.jpg](https://github.com/user-attachments/assets/d060f468-bbd6-4d9e-8bfa-c57a29276b8e)

**Comment by jorgef72** [comment]
Would like this feature, if possible...only part missing from this app
